mentors/scala/r.json

```diff
@@ -14,5 +14,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I really like working in Scala.  It allows me to come up with elegant, functional code that is also concise and type safe."
-  }
+  },
+  {
+      "github_username": "rajeshpg",
+      "name": "Rajesh Pitty",
+      "link_text": null,
+      "link_url": null,
+      "avatar_url": null,
+      "bio": "a passionately curious scala functional programmer"
+    }
```

Comment:
Indentation should be exactly two spaces.

Reply:
fixed the indentation.